src/libs/ReportUtils.js

```diff
@@ -194,18 +194,28 @@ function isSettled(reportID) {
 }
 
 /**
- * Can only delete if the author is this user and the action is an ADDCOMMENT action or an IOU action in an unsettled report
+ * Can only delete if the author is this user and the action is an ADDCOMMENT action or an IOU action in an unsettled report, or if the user is a
+ * policy admin
  *
  * @param {Object} reportAction
+ * @param {String} reportID
  * @returns {Boolean}
  */
-function canDeleteReportAction(reportAction) {
-    return (
-        reportAction.actorEmail === sessionEmail &&
-        ((reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportActionsUtils.isCreatedTaskReportAction(reportAction)) ||
-            (ReportActionsUtils.isMoneyRequestAction(reportAction) && !isSettled(reportAction.originalMessage.IOUReportID))) &&
-        reportAction.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE
-    );
+function canDeleteReportAction(reportAction, reportID) {
```

Comment:
Just a heads-up, this introduced a regression in https://github.com/Expensify/App/issues/21842#issuecomment-1621454928
We missed a case where user is trying to delete a message from Concierge. Even if user is admin, concierge messages shouldn't be deleted. 
We resolved this by adding `reportAction.actorAccountID !== CONST.ACCOUNT_ID.CONCIERGE `condition